packages/workflow/src/Interfaces.ts

```diff
@@ -4,8 +4,6 @@ import type * as express from 'express';
 import type FormData from 'form-data';
 import type { PathLike } from 'fs';
 import type { IncomingHttpHeaders } from 'http';
-import type { OptionsWithUri, OptionsWithUrl } from 'request';
-import type { RequestPromiseOptions } from 'request-promise-native';
```

Comment:
I removed all references to `request-promise-native`, but there is still a lot of code in nodes-base using types from `request`.

Will be a bigger refactor to replace all those, good news is 'request' is not used anymore in other packages.

We should consider if we want to replace the types with `IRequestOptions` or `IHttpRequestOptions` (and `helpers.httpRequest`). 2nd is better but is potentially breaking.